templates/partials/pagination.html

```
@@ -0,0 +1,28 @@
+{{define "pagination"}}
+{{if gt .TotalPages 1}}
+<nav>
+    <ul class="pagination pagination-sm justify-content-center">
+        <li class="page-item{{if eq .CurrentPage 1}} disabled{{end}}">
+            <a class="page-link" href="#"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)